plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveConfig.java

```diff
@@ -1041,4 +1045,17 @@ public boolean isLegacyHiveViewTranslation()
     {
         return this.legacyHiveViewTranslation;
     }
+
+    @Config(ORC_ACID_WRITES_ENABLED_CONFIG_PROPERTY)
+    @ConfigDescription("Enable writes to ORC ACID transactional tables")
```

Comment:
Perhaps add a link to code-comment link to known issue(s)

Reply:
added `//TODO` with issue link